app/controllers/community/meetups/index.js

```diff
@@ -12,6 +12,10 @@ export default Controller.extend({
 
     if (!this.fastboot?.isFastBoot) {
       import('leaflet').then(() => {
+        if (this.isDestroyed || this.isDestroying) {
+          return;
+        }
```

Comment:
I made a temporary fix so that unit tests for `community.meetups.index` controller won't run into `calling set on destroyed object` error.